frontend/src/containers/DonatePage.js

```diff
@@ -1,224 +1,219 @@
-      donationSection = <PublicGroupThanks i18n={i18n} group={group} message={i18n.getString('thankyou')} tweet={tier.tweet} />;
-    } else if (this.state.showUserForm) {
-      donationSection = <PublicGroupSignup {...this.props} save={saveNewUser.bind(this)} />
-    } else {
-      donationSection = <Tiers tiers={tiers} {...this.props} form={donationForm} onToken={donateToGroup.bind(this)} />
+    if (!tier.name || tier.presets && tier.presets.length > 1) {
+      Object.assign(tier, {
+      name: "custom",
+      title: " ",
+      description: tier.description || i18n.getString('defaultTierDescription'),
+      range: [tier.amount, 10000000],
+      tweet
+      });
     }
+    const tiers = [tier];
+    collective.backers = []; // We don't show the backers
 
     return (
-      <div className='DonatePage'>
-        <LoginTopBar />
+      <div className='DonatePage Page'>
         <Notification />
-        <div className='DonatePage-logo' style={{backgroundImage: `url(${group.logo ? group.logo : '/static/images/rocket.svg'})`}}></div>
-        <div className='DonatePage-line1'>Hi! We are <b>{ group.name }</b>. With your support we can</div>
-        {group.mission ? <div className='DonatePage-line2'>{ group.mission }</div> : null}
-        {group.website ? <a className='DonatePage-line3' href={group.website}>{group.website}</a> : null}
-        <div className='DonatePage-donation-container'>{ donationSection }</div>
-        <PublicFooter />
+          <div className='CollectiveDonationFlowWrapper'>
+            <div>
+            { view === 'default' &&
+                <div className="DonatePage-header">
+                  <a href={`/${collective.slug}`}>
+                    <div className='DonatePage-logo' style={{backgroundImage: `url(${collective.logo ? collective.logo : '/static/images/rocket.svg'})`}}></div>
+                  </a>
+                  <div className='DonatePage-line1'>Hi! We are  <a href={`/${collective.slug}`}>{ collective.name }</a>.</div>
+                  {collective.mission ? <div className='DonatePage-line2'>{ collective.mission }</div> : null}
+                </div>
+            }
+            { view === 'default' &&
+              <Tiers
+                tiers={tiers}
+                collective={collective}
+                host={host}
+                donationForm={donationForm}
+                appendDonationForm={appendDonationForm}
+                onToken={donateToGroup.bind(this)}
+                i18n={i18n}
+              />
+            }
+            { view === 'signup' &&
+              <PostDonationSignUp {...this.props} save={saveNewUser.bind(this)} {...this.props} />
+            }
+            { view === 'thankyou' &&
+              <PostDonationThanks i18n={i18n} collective={collective} message={i18n.getString('thankyou')} tweet={tier.tweet} showRelated={false} closeDonationFlow={this.closeDonationFlow.bind(this)} />
+            }
+            { view === 'default' &&
```

Comment:
Would be better to simplify showing 'default' view all in one section and embedding other views as needed.